.drone.yml

```diff
@@ -31,14 +31,14 @@ pipeline:
     repo: docker.digital.homeoffice.gov.uk/drt/drt
     secrets: [ docker_password ]
     tags:
-      - ${DRONE_COMMIT_SHA}
+      - ${DRONE_BUILD_NUMBER}
       - latest
     when:
-      event: [push, tag]
+      event: push
       branch:
         - master
 
-  drt-test-instance:
+  cypress-E2E-test:
```

Comment:
I think we have all lowercase for all the other labels in here.

Reply:
ammended